src/instance-id/instance-id-request.ts

```diff
@@ -94,26 +93,17 @@ export class FirebaseInstanceIdRequestHandler {
         return response;
       })
       .catch((response) => {
-        let error;
-        if (typeof response === 'object' && 'error' in response) {
-          error = response.error;
-        } else {
-          error = response;
-        }
-
+        const error = (typeof response === 'object' && 'error' in response)
+          ? response.error : response;
         if (error instanceof FirebaseError) {
           // In case of timeouts and other network errors, the API request handler returns a
           // FirebaseError wrapped in the response. Simply throw it here.
           throw error;
         }
 
-        let template: string = ERROR_CODES[response.statusCode];
-        let message: string;
-        if (template) {
-          message = `Instance ID "${apiSettings.getEndpoint()}": ${template}`;
-        } else {
-          message = JSON.stringify(error);
-        }
+        const template: string = ERROR_CODES[response.statusCode];
+        const message: string = template
+          ? `Instance ID "${apiSettings.getEndpoint()}": ${template}` : JSON.stringify(error);
```

Comment:
Better for readability to have the ? on the previous line

Reply:
Done